site/src/main/scala/org/typelevel/sbt/TypelevelSitePlugin.scala

```diff
@@ -106,6 +111,12 @@ object TypelevelSitePlugin extends AutoPlugin {
       .value: @nowarn("cat=other-pure-statement"),
     tlSitePreview := previewTask.value,
     Laika / sourceDirectories := Seq(mdocOut.value),
+    laikaConfig := laikaConfig
+      .value
+      .withConfigValue(LinkConfig(apiLinks = tlSiteApiLinks.value.toList.map {
+        case (packagePrefix, baseUri) =>
+          ApiLinks(packagePrefix = packagePrefix, baseUri = baseUri.toString)
+      })),
```

Comment:
Does the order here matter? e.g. what happens if the `cats` prefix comes before the `cats.effect` prefix?

Reply:
Order doesn't matter, [Laika uses longest prefix match](https://github.com/planet42/Laika/blob/605d8a46361abdd696e9e55fd74016e77538e8e7/core/shared/src/main/scala/laika/directive/std/LinkDirectives.scala#L58).